src/UID2.Client/Utils/UID2TokenGenerator.cs

```diff
@@ -6,17 +6,22 @@
 using System.Security.Cryptography;
 using System.Text;
 
-
 namespace UID2.Client.Utils
 {
+    // AU: Since this is essentially test code, we should eventually move it into the test project and probably replace
+    // the confusing combo of "params builder-like + static function" to just be a regular UID2TokenBuilder that produces
+    // a string token on Build()
```

Comment:
Not the whole class is test code - UID2.Client.UID2Client.Encrypt calls UID2.Client.Utils.UID2TokenGenerator.GenerateUid2TokenV4 indirectly for sharing token generation so i don't think this comment is true (we can do refactoring into test-only codes for non v4 generation code but probably can reconsider later)

Reply:
i suggest just remove the comment